<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SystemWidget</class>
 <widget class="QWidget" name="SystemWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>220</width>
    <height>236</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="mainLayout">
   <item row="1" column="0">
    <widget class="QLabel" name="labelMaster">
     <property name="text">
      <string>Master</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QCheckBox" name="Master">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="text">
      <string notr="true"/>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="labelShiftValue">
     <property name="text">
      <string>Shift data</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="ShiftValue">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="text">
      <string notr="true">0b00000000</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="labelShiftActive">
     <property name="text">
      <string>Shift enabled</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QCheckBox" name="ShiftActive">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="text">
      <string notr="true"/>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="labelGain_0">
     <property name="text">
      <string>PGA 0 gain</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLabel" name="Gain_0">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="text">
      <string notr="true">x1</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="labelGain_1">
     <property name="text">
      <string>PGA 1 gain</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLabel" name="Gain_1">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="text">
      <string notr="true">x1</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="labelRam">
     <property name="text">
      <string>Free RAM</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QCheckBox" name="Online">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="text">
      <string notr="true"/>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="labelOnline">
     <property name="text">
      <string>Online</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="labelInterval">
     <property name="text">
      <string>Script interval</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLabel" name="Interval">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="text">
      <string notr="true">0 s</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QProgressBar" name="FreeRam">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="maximum">
      <number>2048</number>
     </property>
     <property name="value">
      <number>1024</number>
     </property>
     <property name="format">
      <string notr="true">%v B (%p%)</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
